pkg/server/options/options.go

```diff
@@ -100,9 +100,10 @@ func NewOptions() *Options {
 		WithClientCert().
 		WithOIDC().
 		WithRequestHeader().
-		// WithServiceAccounts().
+		WithServiceAccounts().
 		WithTokenFile()
 	//WithWebHook()
+	o.GenericControlPlane.Authentication.ServiceAccounts.Issuers = []string{"https://kcp.default.svc"}
```

Comment:
The kube default points to the apiserver service that is always reachable within a cluster, but with kcp lacking services it's not clear we have a good default. Maybe that suggests we should not be enabling bound tokens without an explicit issuer being configured? Or maybe we should provide a default that is not a url (if possible - maybe validation precludes this) to make it clear that the bound tokens issued by default are not intended to be verified via an oidc endpoint?

Reply:
I would leave it on. We can have multiple in the future if we have to change to some other URL for some reason.